src/purchases.ts

```diff
@@ -92,6 +124,15 @@ export default class Purchases {
      */
     public static PURCHASE_TYPE = PURCHASE_TYPE;
 
+    /**
+     * Enum for billing features.
+     *  Currently, these are only relevant for Google Play BillingClient:
```

Comment:
There's an extra space at the beginning of this sentence. Maybe I would do like in the canMakePayments docs:

```
Note: Currently, BillingFeatures are only relevant to Google Play Android users.
```

Some developers might not even know we are abstracting BillingClient